cmd/bootstrapper/generator_test.go

```diff
@@ -46,11 +46,17 @@ var bitcoinResponse2 string
 
 type MeshAPIMock struct{}
 
-func (m *MeshAPIMock) LatestLayer() types.LayerID                        { panic("not implemented") }
-func (m *MeshAPIMock) LatestLayerInState() types.LayerID                 { panic("not implemented") }
-func (m *MeshAPIMock) ProcessedLayer() types.LayerID                     { panic("not implemented") }
-func (m *MeshAPIMock) GetRewards(types.Address) ([]*types.Reward, error) { panic("not implemented") }
-func (m *MeshAPIMock) GetLayer(types.LayerID) (*types.Layer, error)      { panic("not implemented") }
+func (m *MeshAPIMock) LatestLayer() types.LayerID        { panic("not implemented") }
+func (m *MeshAPIMock) LatestLayerInState() types.LayerID { panic("not implemented") }
+func (m *MeshAPIMock) ProcessedLayer() types.LayerID     { panic("not implemented") }
+func (m *MeshAPIMock) GetRewardsByCoinbase(types.Address) ([]*types.Reward, error) {
+	panic("not implemented")
+}
+
+func (m *MeshAPIMock) GetRewardsBySmesherId(types.NodeID) ([]*types.Reward, error) {
```

Comment:
`MeshAPIMock` is only used in a single place and can be replaced with `grpcserver.NewMockmeshAPI(gomock.NewController(tb)` there.